database/createRecreateDatabase.py

```diff
@@ -6,32 +6,50 @@
 import os
 from dotenv import load_dotenv
 import psycopg2
+import argparse
 
 # take environment variables from .env.
 load_dotenv()
+# Example URL to connect to: postgresql://ryubarrett:password@localhost:5432/opmdatabase
+database_url = os.getenv('DATABASE_URL')
 
 # File should be in same directory as python script
-CREATION_SQL_FILE = "RecreateOPMDatabase.sql"
+CREATION_SQL_FILE = "database\RecreateOPMDatabase.sql"
```

Comment:
This assumes there's another `database` directory within database directory, i.e.

`/database/database/RecreateOPMDatabase.sql`

Could you remove the `database\` from these two?

Reply:
We can but if we go from  "database/RecreateOPMDatabase.sql" -> "RecreateOPMDatabase.sql" it runs into a FileNotFoundError, at least on my end.

Reply:
Nevermind, I was in the wrong directory when running the file, so sorry lol

Reply:
NP! You should be able to push a new commit, it'll be pulled into this PR